content/en/tracing/connect_logs_and_traces/ruby.md

```diff
@@ -3,50 +3,35 @@ title: Connecting Ruby Logs and Traces
-    - link: 'tracing/manual_instrumentation'
-      tags: 'Enrich Tracing'
-      text: 'Instrument manually your application to create traces.'
-    - link: 'tracing/opentracing'
-      tags: 'Enrich Tracing'
-      text: 'Implement Opentracing across your applications.'
-    - link: 'tracing/visualization/'
-      tag: 'Use the APM UI'
-      text: 'Explore your services, resources, and traces'
     - link: 'https://www.datadoghq.com/blog/request-log-correlation/'
       tag: 'Blog'
       text: 'Correlate request logs with traces automatically'
 ---
 
-## Manually Inject Trace and Span IDs
+## Trace correlation
 
-In many cases, such as logging, it may be useful to correlate trace IDs to other events or data streams, for easier cross-referencing. The tracer can produce a correlation identifier for the currently active trace via `active_correlation`, which can be used to decorate these other data sources.
+In many cases, such as logging, it may be useful to correlate trace IDs to other events or data streams, for easier cross-referencing.
+
+### For logging in Rails applications
+
+#### Automatic
+
+For Rails applications using the default logger (`ActiveSupport::TaggedLogging`) or `lograge`, you can automatically enable trace correlation injection by setting the `rails` instrumentation configuration option `log_injection` to `true` or by setting environment variable `DD_LOGS_INJECTION=true`:
 
 ```ruby
-# When a trace is active...
-Datadog.tracer.trace('correlation.example') do
-  # Returns #<Datadog::Correlation::Identifier>
-  correlation = Datadog.tracer.active_correlation
-  correlation.trace_id # => 5963550561812073440
-  correlation.span_id # => 2232727802607726424
-  correlation.env # => 'production' (derived from DD_ENV)
-  correlation.service # => 'billing-api' (derived from DD_SERVICE)
-  correlation.version # => '2.5.17' (derived from DD_VERSION)
-end
+# config/initializers/datadog.rb
+require 'ddtrace'
 
-# When a trace isn't active...
-correlation = Datadog.tracer.active_correlation
-# Returns #<Datadog::Correlation::Identifier>
-correlation = Datadog.tracer.active_correlation
-correlation.trace_id # => 0
-correlation.span_id # => 0
-correlation.env # => 'production' (derived from DD_ENV)
-correlation.service # => 'billing-api' (derived from DD_SERVICE)
-correlation.version # => '2.5.17' (derived from DD_VERSION)
+Datadog.configure do |c|
+  c.use :rails, log_injection: true
+end
 ```
 
-#### For logging in Rails applications using Lograge (recommended)
+_Note:_ For `lograge` users who have also defined `lograge.custom_options` in an `initializers/lograge.rb` configuration file, due to the order that Rails loads initializers (alphabetical), automatic trace correlation may not take effect, since `initializers/datadog.rb` would be overwritten by the `initializers/lograge.rb` initializer. To support automatic trace correlation with _existing_ `lograge.custom_options`, use the [Manual (Lograge)](#manual-lograge) configuration below.
```

Comment:
```suggestion
**Note:** For `lograge` users who have also defined `lograge.custom_options` in an `initializers/lograge.rb` configuration file, because Rails loads initializers in alphabetical order, automatic trace correlation may not take effect, since `initializers/datadog.rb` would be overwritten by the `initializers/lograge.rb` initializer. To support automatic trace correlation with _existing_ `lograge.custom_options`, use the [Manual (Lograge)](#manual-lograge) configuration below.
```